Doc/library/mimetypes.rst

```diff
@@ -118,6 +118,18 @@ behavior of the module.
    When *strict* is ``True`` (the default), the mapping will be added to the
    official MIME types, otherwise to the non-standard ones.
 
+.. function:: mimesniff(data)
+
+   Guess the type of given *data*.
+   *data* should be :class:`bytes` or :class:`bytearray`.
+   Returns two string values ``(type, charset)``.
+   This function always returns the most valid type, but charset can be
+   ``None`` if there no proper charset.
```

Comment:
A gotcha: `charset` in web specs is an abbreviation for `character encoding`; I think the long form should be used in the text here.